src/components/newcases.js

```diff
@@ -0,0 +1,140 @@
+      {length > 0 && (
+        <React.Fragment>
+          <h2 className="Level fadeInUp" style={{animationDelay: '0.8s'}}>
+            Rate of New Cases
+          </h2>
+          <div className="Level fadeInUp" style={{animationDelay: '0.8s'}}>
+            <div
+              className={`level-item ${
+                casesMap[3].cases - casesMap[4].cases < 0
+                  ? 'is-green'
+                  : 'is-cherry'
+              }`}
+            >
+              <h5 className="heading">{casesMap[3].date}</h5>
+              <h2 className="title has-text-grey">{casesMap[3].cases}</h2>
+            </div>
+
+            <div
+              className={`level-item ${
+                casesMap[2].cases - casesMap[3].cases < 0
+                  ? 'is-green'
+                  : 'is-cherry'
+              }`}
+            >
+              <h5 className="heading"> </h5>
+              <h1>
+                {casesMap[2].cases - casesMap[3].cases > 0
+                  ? upArrow
+                  : downArrow}
+                {Math.abs(casesMap[2].cases - casesMap[3].cases)}
+              </h1>
+              <h1 className="title has-text-grey">{}</h1>
+            </div>
+
+            <div
+              className={`level-item ${
+                casesMap[2].cases - casesMap[3].cases < 0
+                  ? 'is-green'
+                  : 'is-cherry'
+              }`}
+            >
+              <h5 className="heading">{casesMap[2].date}</h5>
+              <h2 className="title has-text-success">{casesMap[2].cases} </h2>
+            </div>
+
+            <div
+              className={`level-item ${
+                casesMap[1].cases - casesMap[2].cases < 0
+                  ? 'is-green'
+                  : 'is-cherry'
+              }`}
+            >
+              <h5 className="heading has-text-info">{} </h5>
+              <h1 className="has-text-info">
+                {casesMap[1].cases - casesMap[2].cases > 0
+                  ? upArrow
+                  : downArrow}
+                {Math.abs(casesMap[1].cases - casesMap[2].cases)}
+              </h1>
+              <h1 className="title has-text-grey">{}</h1>
+            </div>
+
+            <div
+              className={`level-item ${
+                casesMap[1].cases - casesMap[2].cases < 0
+                  ? 'is-green'
+                  : 'is-cherry'
+              }`}
+            >
+              <h5 className="heading">{casesMap[1].date}</h5>
+              <h2 className="title has-text-info">{casesMap[1].cases}</h2>
+            </div>
+
+            <div className="level-item is-blue">
+              <h5 className="heading"> </h5>
+              <h1>
+                {casesMap[0].cases - casesMap[1].cases > 0
+                  ? upArrow
+                  : downArrow}
+                {Math.abs(casesMap[0].cases - casesMap[1].cases)}*
+              </h1>
+              <h1 className="title has-text-grey">{}</h1>
+            </div>
```

Comment:
All this code is repetitive. You can use a for loop and simply for this part. Since cases data is in an array.

Reply:
added for loop for this.